crates/fluvio-connector-package/src/context.rs

```diff
@@ -0,0 +1,11 @@
+use std::collections::HashMap;
+
+use serde::Serialize;
+
+/// Context for the template engine
+/// This is the data that is available to the template engine
+/// when it is rendering the template.
+#[derive(Serialize)]
+pub(crate) struct Context {
+    pub(crate) secrets: HashMap<String, String>,
```

Comment:
Shouldn't secret be using SecretString?

Reply:
This is the actual context used to render the template. So, for template engine we need that this serializes to:
```json
{
  "secrets": {
      "foo": "bar"
   }
}

So, templeta engine knows that `${{ secrets.foo }}` should be replaced by `bar`

   ```